helm/prometheus-agent/values.yaml

```diff
@@ -45,6 +45,8 @@ prometheus-agent:
   version: ""
 
   serviceMonitor:
+    enabled: true
```

Comment:
config changes are coming from https://github.com/giantswarm/observability-bundle/blob/aa252ceda12017aff2002dcee3170024d06ef386/helm/observability-bundle/values.yaml#L59C11-L88C34 so we can clean up the bundle